msmtools/util/sorted_schur.py

```diff
@@ -352,7 +352,11 @@ def sorted_krylov_schur(P, m, z='LM'):
     # WE NEED REAL VECTORS! G-PCCA and PCCA only work with real vectors!!
     # We take the sequence of 1-D arrays and stack them as columns to make a single 2-D array.
     Subspace = np.column_stack([x.array for x in E.getInvariantSubspace()])
-    
+
+    R = E.getDS().getMat(SLEPc.DS.MatType.A)
```

Comment:
Have you tested that this really gives you the quasi-triangular matrix? (e.g. via comparison with R given back from brandts)

Reply:
Mike has implemented this, I will test it and get back to you!